Kernel/Time/TimeManagement.cpp

```diff
@@ -137,6 +139,7 @@ u64 TimeManagement::uptime_ms() const
     return ms;
 }
 
+#if ARCH(I386) || ARCH(X86_64)
```

Comment:
Might be nicer to move the `#if ARCH` into the function so you don't have to declare them twice?